modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/cart_mini/view.jsp

```diff
@@ -105,17 +105,26 @@ request.setAttribute("view.jsp-portletURL", portletURL);
 
 				<%
 				CPDefinition cpDefinition = commerceOrderItem.getCPDefinition();
+
+				String cpInstanceCDNURL = commerceCartContentMiniDisplayContext.getCPInstanceCDNURL(commerceOrderItem);
 				%>
 
 				<liferay-ui:search-container-column-text
 					cssClass="col-1 thumbnail-section"
 				>
 					<span class="sticker sticker-xl">
 						<span class="sticker-overlay">
-							<liferay-adaptive-media:img
-								class="sticker-img"
-								fileVersion="<%= commerceCartContentMiniDisplayContext.getCPInstanceImageFileVersion(commerceOrderItem) %>"
-							/>
+							<c:choose>
+								<c:when test="<%= Validator.isNotNull(cpInstanceCDNURL) %>">
```

Comment:
Ehi @brianikim and @alessiorendina why this changes weren't made also in the react minicart component?

Reply:
@andrea-ale-sbarra , could you direct me to the correct file you are referencing?

Reply:
@andrea-ale-sbarra , https://github.com/liferay-commerce/liferay-portal/blob/4d087767da18b2fbdeea48b705a9dc12b623e594/modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItem.js#L55 , the change you are requesting is not necessary because that code utilizes the back-end code of https://github.com/liferay-commerce/liferay-portal/pull/3044/commits/329a20067e39fdf6251df31b63f3d5958731b446 which fixes the issue. See use-case: https://github.com/liferay-commerce/liferay-portal/blob/4d087767da18b2fbdeea48b705a9dc12b623e594/modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/dto/v1_0/CartItemDTOConverter.java#L85

Reply:
Alright thank you @brianikim